docs/HowToFAQ/FAQSplitHere.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+---
+
+## Question
+
+How do `{:split_here}` and `{:focus}` work to divide up a verification condition?
+
+## Answer
+
+Analysis of a method for the purpose of verification identifies a large number of assertions that must be proved.
+These assertions are grouped into batches.
+
+One default way of operating is to combine all assertions into one batch, make one verification condition out of it,
+and have the prover tackle the whole thing at once. This method allows the prover to use intermediate results to 
```

Comment:
Should we call an SMT solver a prover, just because we're using it to prove things?

Reply:
Done